src/patternfly/components/Form/form.scss

```diff
@@ -7,6 +7,9 @@
   --pf-c-form--m-horizontal__group-label--md--GridColumnGap: var(--pf-global--spacer--md);
   --pf-c-form--m-horizontal__group-control--md--GridColumnWidth: 1fr;
 
+  // limit width
+  --pf-c-form--m-limit-width--md--Width: #{pf-size-prem(500px)};
```

Comment:
Should this be `--pf-c-form--MaxWidth: auto`? Then `--pf-c-form--MaxWidth` can be updated inline on the parent `form`.